src/Tizen.NUI.Components/Controls/RadioButton.cs

```diff
@@ -38,7 +38,10 @@ public class RadioButton : SelectButton
         /// Creates a new instance of a RadioButton.
         /// </summary>
         /// <since_tizen> 8 </since_tizen>
-        public RadioButton() : base() { }
+        public RadioButton() : base()
+        {
+            EnableToggleSelected = false;
```

Comment:
The description of `EnableToggleSelected` is 

> EnableToggleSelected is used to indicate if SelectButton is selected and unselected by toggling.

But I think the RadioButton is a control that can be selected/unselected by toggling.
I think the word `toggle` is confusing for this.

Reply:
Thank you for your comment.

About the meaning of "toggle", I meant as follows if EnableToggleSelected is true.
- SelectButton becomes "selected" if its state is "unselected" when it is clicked.
- SelectButton becomes "unselected" if its state is "selected" when it is clicked.

Could you please share the alternative words instead of "toggle" with me if you have any idea?

About supporting selected/unselected by toggling in RadioButton, I think as follows.

- As far as I know, radio widget is not selected/unselected by toggling because normally radio widget group wants one radio widget is selected all the time. (Selecting no radio widget is not an option.)
  e.g. In Fonts of Settings, multiple radio widgets are displayed for multiple font types and one font type should be selected. (e.g. default font)

- Currently, it appears that NUI's RadioButton is not selected/unselectd by toggling as well.
  (The RadioButton is not unselected by toggling although RadioButton blinks by toggling)

Because of the above reason, I set EnableToggleSelected false in RadioButton's constructor.
I think it would be better if RadioButton does not support selected/unselected by toggling by default.

Reply:
답변 감사합니다~
토글이 한 방향(unselect에서 select로)으로는 항상 가능하다는 점이 제가 헷갈리는 이유인것 같습니다.
그룹에 속해 있을때는, 다른 버튼을 선택함으로 현재 버튼을 select에서 unselect 상태로 만들 수 있기도 합니다.
RadioButton은 단독으로 toggle button 이라고 부르긴 힘들지만, 그렇다고 toggle이 아예 불가능하지도 않기 때문에 애매한 것 같아요.
개발자마다 의견이 다르겠지만, Android의 [RadioButton](https://developer.android.com/guide/topics/ui/controls/radiobutton)도 radioButton.toggle()로 select 상태로 변경하는 것이 가능합니다. 즉 toggle이라는 단어가 radio button에서도 의미를 가진다고 생각합니다.

현재 패치에 어울리는 더 나은 단어를 찾아보겠습니다.
감사합니다.

Reply:
답변 확인이 늦어져 죄송합니다.

제가 생각한 토글은 터치(클릭)할 때마다 오브젝트의 상태를 전환하는 기능이었습니다. (Selected -> Unselected / Unselected -> Selected)
그래서 일반적인 RadioButton의 경우, 터치(클릭)할 때마다 항상 오브젝트의 상태가 전환되지는 않고, Unselected인 상태에서만 Selected로 전환되는 모습을 보여주기에 토글 속성을 갖고 있지 않다고 생각했습니다. 그래서 EnabledToggleSelected를 false로 설정하였습니다.

하지만 말씀해주신 것처럼 토글 속성이 없을 때, 터치(클릭)을 통해 Selected -> Unselected는 허용되지 않지만, Unselected -> Selected는 허용하는 것이 혼란을 줄 수도 있을 것 같습니다. 특히 말씀해주신 Android의 toggle()과 같은 함수가 추가된다면, EnabledToggleSelected는 이해하기 쉽지 않은 함수가 될 수도 있을 것 같습니다.

저도 더 나은 다른 단어를 찾아보고 다시 의견 구할 수 있도록 하겠습니다.
감사합니다.